apiary.apib

```diff
@@ -1349,6 +1349,7 @@ Get a list of deals.
     + Attributes (object)
         + filter (object, optional)
             + ids: `3f507137-e599-41d4-8625-0e2adba0be4c`,`ae97f6d3-0bb4-43d3-b13d-f06069d07e98` (array[string], optional)
+            + term: `Interesting deal` (string, optional) - Filters on the title, reference and customer's name
```

Comment:
I think `quotation number` is a more correct domain term than `reference`, right?

Summoning the deals team: @dietrichm @steftrenson

Reply:
Ah, I went with `reference` as I assumed that's the key it's refering to

Reply:
Going to double check if that's the same.

Reply:
Yep, it's the same!